stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/CommandMentionListItemAdapter.java

```diff
@@ -53,57 +48,18 @@ public long getItemId(int position) {
     public View getView(int position, View convertView, ViewGroup parent) {
         ViewBinding binding;
         if (convertView == null) {
-            if (isCommand) {
-                convertView = layoutInflater.inflate(R.layout.stream_item_command, null);
-                binding = StreamItemCommandBinding.bind(convertView);
-            } else {
-                convertView = layoutInflater.inflate(R.layout.stream_item_mention, null);
-                binding = StreamItemMentionBinding.bind(convertView);
-            }
```

Comment:
How come we removed one of these cases? Was it simply unused?

Reply:
It wasn't used anywhere. This adapter seems to be implemented to be used in a generic way in different places prematurely, and it wasn't.